### ISSUE TITLE
[FEATURE REQUEST] scalar elemental embeddings

### ISSUE THREAD

**Issue by sgbaird** [lantunes/skipatom#10]
How difficult would it be to get a scalar (rather than vector) version of SkipAtom? I'm interested in using this with https://github.com/sparks-baird/chem_wasserstein and https://github.com/sparks-baird/mat_discover

**Comment by lantunes** [comment]
I'm actually not familiar with scalar elemental embeddings. I suppose that we'd like to associate a single scalar with each element? Is that equivalent to learning element vectors that have only a single component? Is there any literature you can point me to that discusses this?

**Comment by sgbaird** [comment]
Yes, that's right - essentially a "replacement" for the atomic number scale. Mendeleev number, electronegativity, modified Pettifor, are each examples. ElMD manuscript covers some additional scales https://pubs.acs.org/doi/full/10.1021/acs.chemmater.0c03381. 

Modified Pettifor scale https://iopscience.iop.org/article/10.1088/1367-2630/18/9/093011/meta

Some follow-up discussion worth perusing https://github.com/lrcfmd/ElMD/issues/27

Another option is to do dimensionality reduction on the SkipAtom vector embeddings, e.g. via DensMAP. Lmk what you think!

ElMD was also implemented for vector elemental embeddings, but it's computationally much more expensive to calculate Earth Mover's Distance between 2D distributions. A certain "trick" no longer applies.

Curious to hear your thoughts.

**Comment by sgbaird** [comment]
I was reading back through the modified Pettifor manuscript, and I think it would make sense to think of it like a 1D periodic table. Might try to do dimensionality reduction on the SkipAtom vectors.
